### PR TITLE
fix already released version and duplicated dependencies/plugins

### PR DESCRIPTION
**Pull request by mareknovotny**
We already have released version 1.0 in Maven Central, but the latest tuning and externalizing from kiegroup/droolsjbpm-integration repository kept a outstanding things:

- master should have SNAPSHOT version
- release a 1.1 version to have available the latest content

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -12,7 +12,7 @@
   <groupId>org.kie</groupId>
   <artifactId>kie-gwthelper-maven-plugin</artifactId>
   <packaging>maven-plugin</packaging>
-  <version>1.0</version>
+  <version>1.1-SNAPSHOT</version>
```
Comment (gitgabrio) [inline pom.xml:15]: @mareknovotny 
Due to some enforcer rule we can not have SNAPSHOT versions on plugins

**Review by gitgabrio — CHANGES_REQUESTED** [review]
@mareknovotny 
For me it is not an issue to change the plugin version - of course it must also be updated in bom. Beside that, pls read comment about version

File: pom.xml (hunk 1 above)
Comment (mareknovotny) [reply pom.xml:15]: yes, no problem. We should have referenced in other repositories only the released versions

So the master is development stable branch, and the released version is used in other kiegroup repositories.

Comment (mareknovotny) [reply pom.xml:15]: just to be sure the change to 1.1 version needs to happen in release of 1.1 by tagging that version and release it into Maven Central. Then you can update the version in other repository

**Review by gitgabrio — APPROVED** [review]

**Pull request merged**